webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala

```diff
@@ -473,7 +473,8 @@ class Boot extends Loggable {
     LiftRules.ajaxEnd = Full(() => LiftRules.jsArtifacts.hide("ajax-loader").cmd)
     LiftRules.ajaxPostTimeout = 30000
 
-    LiftRules.maxMimeFileSize = 10 * 1024 * 1024
+    // Same as default LiftRules.maxMimeSize
+    LiftRules.maxMimeFileSize = 8 * 1024 * 1024
```

Comment:
:question:  I couldn't find a reason for the previous value

Reply:
Since it links to a commit from Rudder 5.0 where we merged back inventories into rudder, I imagine it was the max of the value in the two projects (and the other wasn't using lift, so perhaps it was just a misuse of `maxMimeFileSize` in place of `maxMimeSize`). 
It looks good to correct that to avoid fasle expectation on the limits.